src/SDKs/HDInsight/Management.HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Resources;
+using System.Runtime.CompilerServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Azure .NET SDK")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+[assembly: AssemblyTitle("Microsoft Azure HDInsight Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure HDInsight.")]
+[assembly: AssemblyVersion("3.1.0.0")]
```

Comment:
@wawon-msft you don't have to bump up the AssemblyVersion.
Only bumping AssemblyFileVersion to 3.0.0.0 will be enough.
I just now realized you have previous versions that you use to publish using Hyak.
So all you need is AssemblyFileVersion and AssemblyVersion to 3.0.0.0